pages/home.jsx

```diff
@@ -5,15 +5,156 @@ var Link = Router.Link;
 var HeroUnit = require('../components/hero-unit.jsx');
 var Blockquote = require('../components/blockquote.jsx');
 var Illustration = require('../components/illustration.jsx');
+var IconLinks = require('../components/icon-links.jsx');
+var IconLink = require('../components/icon-link.jsx');
 
 var CaseStudies = React.createClass({
   render: function() {
     return (
-      <div className="case-studies">
-        <Blockquote className="primary-quote" author="Maurya C. New York, United States"
-            imgSrc="/img/pages/home/maurya-nyc.png" imgSrc2x="/img/pages/home/maurya-nyc@2x.png" imgAlt="Maurya NYC Quote">
-          <p>Web literacy is about more than coding - it's about how you can be a better web citizen.</p>
-        </Blockquote>
+      <div className="row">
+        <div className="col-sm-offset-2 col-sm-8 col-md-offset-2 col-md-8 col-lg-offset-2 col-lg-8">
+          <div>
+            <Blockquote author="Maurya C. New York, United States"
+                imgSrc="/img/pages/home/maurya-nyc.png" imgSrc2x="/img/pages/home/maurya-nyc@2x.png" imgAlt="Maurya NYC Quote">
+              <p>"Web literacy is about more than coding - it's about how you can be a better web citizen."</p>
+            </Blockquote>
+          </div>
+        </div>
+      </div>
+    );
+  }
+});
+
+var FeaturedPost = React.createClass({
+  propTypes: {
+    data: React.PropTypes.object.isRequired
+  },
+  render: function() {
+    var publishedDate = new Date(this.props.data.publishedDate);
+    var month = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"][publishedDate.getMonth()];
```

Comment:
I'm wondering if we should just use a library like [moment.js](http://momentjs.com/) here? That way we won't have to write unit tests for this month logic, and [i18n](http://momentjs.com/docs/#/i18n/) should hopefully be easier too (when we get to that).